cypress/components/card/card.cy.tsx

```diff
@@ -1,6 +1,8 @@
-/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable jest/valid-expect */
+/* eslint-disable jest/valid-expect-in-promise */
```

Comment:
```suggestion
/* eslint-disable jest/valid-expect, jest/valid-expect-in-promise */
```

Reply:
Approved providing these jest disable comments will be chained.